component/core/src/main/java/org/exoplatform/social/core/space/impl/SpaceServiceImpl.java

```
@@ -415,13 +416,17 @@ public Space createSpace(Space space, String creator) {
    */
   public Space createSpace(Space space, String creator, String invitedGroupId) {
     // Add creator as a manager and a member to this space
+    Pattern pattern = Pattern.compile("^([\\p{L}\\s\\d\'&]+[\\s]?)+$");
     String[] managers = space.getManagers();
     String[] members = space.getMembers();
     managers = (String[]) ArrayUtils.add(managers,creator);
     members = (String[]) ArrayUtils.add(members,creator);
     space.setManagers(managers);
     space.setMembers(members);
-    
+    if (space.getDisplayName().length() > 200 || !pattern.matcher(space.getDisplayName()).matches()){
+      LOG.error("Error while creating the space"+ space.getPrettyName());
```

A log is not sufficient, it should throw an exception to stop the creation (no need to try to create the space if we know it will fail). The exception message (one per check) should explain what is the creation failure reason.